### PR TITLE
Add Case Studies Infrastructure

### PR DESCRIPTION
**Pull request by hschafer**
Added the notion of a "case study" to the table of contents and added prev/next buttons to navigate.

Also a slight CSS upgrade of the header to make the experience cleaner.

### CODE REVIEW

**Comment by hschafer** [comment]
Sometimes I want to avoid work so I write useless frontend code to avoid responsibility 😤

File: website/components/layout.js [hunk 1]
```diff
@@ -8,14 +9,72 @@ import Navbar from "react-bootstrap/Navbar";
...
+    <div key={caseStudy.caseStudy}>
+      <NavDropdown.Item>
+        <button
+          onClick={onClick}
+          aria-controls={`case-study-${caseStudy.caseStudy}`}
+          aria-expanded={open}
+          className={styles.unbutton}
+        >
+          {" "}
+          🌟 {caseStudy.title}
```
Comment (Rockett8855) [inline website/components/layout.js:48]: The color of the star kinda clashes with the background imo, I would do no bullet point and make the text not italic, unless the star has signficance.

**Comment by hschafer** [comment]
> Here are some thing i noticed/think should change.
> 
> 1. No next chapter on the home page if possible.
> 2. The text is really hard to read with that shade of purple, I'd either make it a darker purple with lighter text, or a lighter background with black text.
> 3. The BM tag is aligned to the center of the page not the column, idk why this is.

Fixed all of these! The BM tag had an issue with CSS specificity that I introduced in the CSS update 😭

**Comment by hschafer** [comment]
Whoops, forgot to actually push the commits :P

**Pull request merged**